int32 ID
latlon latlon
float64 heading_degrees
float64 speed_kmph
